zingolib/src/wallet/notes/transparent.rs

```diff
@@ -53,6 +54,47 @@ impl OutputInterface for TransparentOutput {
     fn pending_spent_mut(&mut self) -> &mut Option<(TxId, u32)> {
         &mut self.unconfirmed_spent
     }
+
+    fn transaction_record_to_outputs_vec(transaction_record: &TransactionRecord) -> Vec<&Self> {
+        let mut vecref = vec![];
+        for output in &transaction_record.transparent_outputs {
+            vecref.push(output);
+        }
+        vecref
```

Comment:
Fluent is simpler here
```suggestion
        transaction_record.sapling_notes.iter().collect()
```

Reply:
AWESOME